obs-browser-source.cpp

```diff
@@ -134,14 +134,21 @@ bool BrowserSource::CreateBrowser()
 		if (hwaccel) {
 			obs_enter_graphics();
 			tex_sharing_avail = gs_shared_texture_available();
+#if CHROME_VERSION_BUILD >= 4430 && defined(__APPLE__) && defined(__arm64__)
+			// CEF 4430 doesn't seem to work correctly when init'd with shared texture on arm64:
+			//  - we never see OnAcceleratedPaint called
+			//  - everything is black
+			// so we'll disable that for the time being
+			tex_sharing_avail = false;
+#endif
```

Comment:
Just noticed this was within `obs_enter_graphics()`.  Though it is just one extra assignment, we try to do as little possible inside the graphics context as possible.  This looks like it can safely be done after `obs_leave_graphics()`.  Or you could rearrange the ifdefs, but it's probably less complicated to just move this one.